src/Common/src/TypeSystem/Common/MetadataFieldLayoutAlgorithm.cs

```diff
@@ -873,7 +876,7 @@ public override DefType ComputeHomogeneousFloatAggregateElementType(DefType type
             if (type.IsWellKnownType(WellKnownType.Double) || type.IsWellKnownType(WellKnownType.Single))
                 return type;
 
-            for (;;)
+            for (; ; )
```

Comment:
No idea why this formatting changed, looks like VS did that by itself. I can revert the change if you find it annoying or contrary to CoreRT coding style.